cpp/log/cert.h

```diff
@@ -272,6 +277,26 @@ class TbsCertificate {
   DISALLOW_COPY_AND_ASSIGN(TbsCertificate);
 };
 
+inline std::ostream& operator<<(std::ostream& out,
+                                const Cert::Status& status) {
+  switch (status) {
+    case Cert::Status::TRUE:
+      out << "TRUE";
+      break;
+    case Cert::Status::FALSE:
+      out << "FALSE";
+      break;
+    case Cert::Status::ERROR:
+      out << "ERROR";
+      break;
+    case Cert::Status::UNSUPPORTED_ALGORITHM:
+      out << "UNSUPPORTED_ALGORITHM";
+      break;
```

Comment:
That sounds like the best way of doing it. I really must get used to all these post-2006 compiler innovations I get to enjoy now!